tests/migration/migration.go

```diff
@@ -3593,3 +3590,7 @@ func runStressTest(vmi *v1.VirtualMachineInstance, vmsize string, stressTimeoutS
 		time.Sleep(15 * time.Second)
 	}
 }
+
+func newVirtualMachineInstanceWithFedoraContainerDisk() *v1.VirtualMachineInstance {
```

Comment:
It is prefered to do this in a separate commit in order to reason for it.

Is there an explanation why this is not inlined now?

Reply:
If we try to change it from function pointer to `vmi *v1.VirtualMachineInstance`
and invoke the functions that are there,
i.e `	Entry("[sig-storage][test_id:2226] with ContainerDisk", libvmi.NewFedora(libnet.WithMasqueradeNetworking()...), false),`

I get the following error, since it is part of the `Entry`
```
Assertion or Panic detected during tree construction
tests/migration/migration.go:2161
  Ginkgo detected a panic while constructing the spec tree.
  You may be trying to make an assertion in the body of a container node
  (i.e. Describe, Context, or When).

  Please ensure all assertions are inside leaf nodes such as BeforeEach,
  It, etc.

...

  Learn more at: http://onsi.github.io/ginkgo/#no-assertions-in-container-nodes
```

Reply:
I do not know what you tried, but you should be able to inline this easily.
`func () *v1.VirtualMachineInstance  {return libvmi.NewFedora(libnet.WithMasqueradeNetworking()...)}`

Reply:
tried to invoke it
ok to inline like that, sure

Reply:
tried to invoke it
ok to inline like that, sure it is possible
but it happens twice, so to use it twice ?
i can also keep it lambda ...